packages/loot-core/migrations/1679728867040_rules_conditions.sql

```diff
@@ -0,0 +1,5 @@
+BEGIN TRANSACTION;
+
+ALTER TABLE rules ADD COLUMN conditionsOp TEXT DEFAULT 'and';
```

Comment:
same thing here about snake_case

Reply:
bump

Reply:
Sorry but took more time than expected to properly map `conditionsOp` to `conditions_op`  😅 
Took me a bit to figure out what I thought was the best place (inside `fromJS` and `toJS`) and realise it should also return partial data without undefined.